packages/cache-utils/src/fs.js

```diff
@@ -30,9 +29,4 @@ const getSrcGlob = async function(src) {
   return srcBasename
 }
 
-// Remove a cached file/directory from the filesystem
-const removeCacheFile = async function(cachePath) {
```

Comment:
I appreciate the removal of this wrapper — less indirection makes this much easier to read 💜

Reply:
Thanks @jlengstorf